txpipe/diagnostics.py

```diff
@@ -372,7 +373,12 @@ def plot_g_histogram(self):
                 R = calculate_shear_response(data['mcal_g1_1p'][qual_cut],data['mcal_g1_2p'][qual_cut],data['mcal_g1_1m'][qual_cut],data['mcal_g1_2m'][qual_cut],
                                                   data['mcal_g2_1p'][qual_cut],data['mcal_g2_2p'][qual_cut],data['mcal_g2_1m'][qual_cut],data['mcal_g2_2m'][qual_cut],delta_gamma)
                 
-                g1, g2 = apply_metacal_response(R, S, data['mcal_g1'][qual_cut][w1], data['mcal_g2'][qual_cut][w1])
+                if self.config['shear_catalog_type']=='metacal':
+                    g1, g2 = apply_metacal_response(R, S, data['mcal_g1'][qual_cut][w1], data['mcal_g2'][qual_cut][w1])
+                elif self.config['shear_catalog_type']=='nonmetacal':
```

Comment:
Since it's currently calling `apply_lensfit_calibration` could you change the param name to just `lensfit`?